docs/aws/aws.md

```diff
@@ -60,7 +60,13 @@ Ensure that the AWS credentials being used have the following permissions. (This
         "ec2:DescribeInstanceTypes",
         "ec2:GetConsoleOutput",
         "ec2:TerminateInstances",
-        "ec2:DescribeVpcAttribute"
+        "ec2:DescribeVpcAttribute",
+        "ec2:CreateSecurityGroup",
```

Comment:
As SREs running manually the checks, we don't have (and don't want/won't have Create/Delete SG permissions).
This should be clarified in the docs. 
As a SRE, the expected flow is to specify the security group on the command line (as the default SG may not be suitable and does not correspond to what is used by OCP anyway). As discussed, the verifier shouldn't run in case no security group is provided and nicely error. 
For the node, we have a card already tracking the implementation of smart retrieve of the SG to be used to avoid the user to have to connect to AWS and retrieve it by him/herself (tracked in OSD-12250).

Reply:
Added it as part of the latest PR